### PR TITLE
Move to botkit. Test env setup

### PR DESCRIPTION
**Pull request by DDKatch**
#34 

Screen Shot 2019-05-23 at 2 30 50 PM

I've used https://github.com/gratifyguy/botkit-mock and `mocha` here. Decided to go with mocha because [this library](https://github.com/gratifyguy/botkit-mock) uses it for self testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,11 @@ Then bot will be available in slack.
 Deployment is run automatically after merging changes to the master branch.
 
 Mona deployed on Heroku.
+
+## Testing
+
```
Comment (AleksSenkou) [inline README.md:21]: Thoughts about describing which modules we use for testing?

**Review by AleksSenkou — APPROVED** [review]
@DDKatch great to have it, thank you!

Comment:
`package-lock.json` is a cool thing to keep in a repo. 
Issue: have to be added as a separate commit and a separate PR (good practice to follow)

**Comment by DDKatch** [comment]
@AleksSenkou 🤔 interesting idea. But why is it better to go with separate PR for that?

File: README.md (hunk 1 above)
Comment (DDKatch) [reply README.md:21]: going to add these notes

**Comment by AleksSenkou** [comment]
@DDKatch because of almost all the time PR has a domain list of changes and adding not related auto-generated files confuse

**Comment by DDKatch** [comment]
@AleksSenkou as you can see package-lock.json file was added in **move-to-botkit** branch. According to #37 PR (going to close it soon) all these 3k+ changes were caused by 2 packages, which I installed in separate commits, so I think there's no issue now.

**Pull request merged**